packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -17,15 +17,16 @@ import {
   hasTail,
   isEarliestUnread
 } from '@audius/common'
+import type { ChatMessage } from '@audius/sdk'
 import { useFocusEffect } from '@react-navigation/native'
-import { View, Text, Image } from 'react-native'
 import type { FlatList as RNFlatList } from 'react-native'
+import { View, Text, Image, FlatList } from 'react-native'
 import { useDispatch, useSelector } from 'react-redux'
 
 import WavingHand from 'app/assets/images/emojis/waving-hand-sign.png'
 import IconKebabHorizontal from 'app/assets/images/iconKebabHorizontal.svg'
 import IconSend from 'app/assets/images/iconSend.svg'
-import { TextInput, Screen, FlatList, ScreenContent } from 'app/components/core'
+import { TextInput, Screen, ScreenContent } from 'app/components/core'
```

Comment:
Don't we actually want to be using our Flatlist and not RN's?

Reply:
yeah def, i think i removed to test if our impl was messed up, will revert